### PR TITLE
chore: update to babylon lab

### PR DESCRIPTION
**Pull request by jrwbabylonlab**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "format": "prettier --check .",
     "format:fix": "prettier --write .",
     "clean": "rm -r node_modules",
-    "build-docker": "docker build -t babylonchain/simple-staking .",
-    "clean-docker": "docker rmi babylonchain/simple-staking 2>/dev/null; true",
+    "build-docker": "docker build -t babylonlab/simple-staking .",
+    "clean-docker": "docker rmi babylonlab/simple-staking 2>/dev/null; true",
```
Comment (vitsalis) [inline package.json:14]: Should be `babylonlabs`

**Review by vitsalis — APPROVED** [review]
minor comment

File: .env.example [hunk 2]
```diff
@@ -1,4 +1,4 @@
 NEXT_PUBLIC_MEMPOOL_API=https://babylon.mempool.space
-NEXT_PUBLIC_API_URL=https://staking-api.staging.babylonchain.io
+NEXT_PUBLIC_API_URL=https://staking-api.testnet.babylonchain.io
```
Comment (gbarkhatov) [inline .env.example:2]: API will use labs domain?

**Review by gbarkhatov — APPROVED** [review]
`terms.tsx`
- links
`Footer.tsx`
- links
- email
`layout.tsx` 
- links
`.env.example`
- API links

have mentions of Chain. I doubt that all of them should use Labs, but we need to check out

**Comment by gbarkhatov** [comment]
@jrwbabylonlab also I think it's "Babylon Labs, Ltd." not "BabylonLabs Ltd." (space)

**Comment by vitsalis** [comment]
> yep. none of the non-github links can be updated. will need devops to change the url first.

We shall be getting an update on this in the following days. Suggest we merge this then

**Pull request merged**